index.src.html

```diff
@@ -615,7 +622,15 @@ <h4 id="secure-context-restriction">Secure context restriction</h4>
 
           1.  If |connection|'s [=connection/IP address space=] is
               [=IP address space/less public=] than |clientAddressSpace|, then
-              return a [=network error=].
+              show a [=permission prompt=].
```

Comment:
I thought we always block clients in insecure contexts, and only show a permission prompt to clients in secure contexts when the "connection" is in HTTP.

The [explainer](https://github.com/WICG/private-network-access/blob/main/permission_prompt/explainer.md) says
```
This would mean that websites that wish to speak to private network devices must be served over HTTPS. The target device, however, would not have to serve HTTPS. It would only need to respond correctly to CORS preflights, and maybe to some kind of simple identification request.
```